CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Description of the upcoming release here.
 
 ### Changed
 
+- [#1318](https://github.com/FuelLabs/fuel-core/pull/1318): Modified block synchronization to use asynchronous task execution when retrieving block headers. Removed the `--sync-max-header-batch-requests` CLI argument, and renamed `--sync-max-get-txns` to `--sync-block-stream-buffer-size` to better represent the current behavior in the import.
```

Comment:
After renaming of the CLI argument, it is a breaking change

Reply:
Yes, I will move the CLI related part of the changelog to a breaking entry.